components/coding/studentPortal/AssignmentInputBox.tsx

```diff
@@ -33,6 +34,17 @@ export const AssignmentInputBox: React.FC<AssignmentInputBoxProps> = ({
     }
   );
 
+  const validateInput = () => {
+    if (
+      !submissionInput.includes("codesandbox") ||
+      !submissionInput.includes("https")
+    ) {
+      alert("Please submit a valid submission link.");
+      return false;
```

Comment:
how about we disable the button if the validateInput function returns false.. could do something like in the goals feature if the text exceeds the number of characters

Reply:
great idea!